templates/intro.md

```diff
@@ -1,56 +1,49 @@
-will be based on the text you had typed into the Add Notes window. If
-you opened the window while editing a note, the preview will be based on
-the content of that note. If you opened the window from Tools → Manage
-Note Types, Anki will display each field’s name in parentheses in place
-of content.
-
-At the top right of the window is an Options button that gives you
-options to rename or reorder the cards, as well as the following two
-options:
-
--   The 'Deck Override' option allows you to change the deck that cards
-    generated from the current card type will be placed into. By
-    default, cards are placed into the deck you provide in the Add Notes
-    window. If you set a deck here, that card type will be placed into
-    the deck you specified, instead of the deck listed in the Add Notes
-    window. This can be useful if you want to separate cards into
-    different decks (for instance, when studying a language, to put
-    production cards in one deck and recognition cards in another). You
-    can check which deck the cards are currently going to by choosing
-    Deck Override again.
-
--   The 'Browser Appearance' option allows you to set different (perhaps
-    simplified) templates for display in the Question and Answer columns
-    of the browser; see [browser appearance](templates/styling.md#browser-appearance) for more
-    information.
+# Modelos de Cartões
+
+Os modelos de cartões contam ao Anki quais campos devem aparecer na frente e no
+verso do seu cartão e controlam quais cartões serão gerados quando certos campos têm texto.
+Ajustando seus modelos de cartões, você pode alterar o design e o estilo de muitos
+dos seus cartões de uma vez só.
+
+Modelos de cartões são explicados em alguns dos vídeos introdutórios (em inglês) a seguir:
+
+-   [Mudando a ordem de cartões](http://www.youtube.com/watch?v=DnbKwHEQ1mA&yt:cc=on);
+
+-   [Estilizando cartões](http://www.youtube.com/watch?v=F1j1Zx0mXME&yt:cc=on);
+
+-   [Digitando a resposta](http://www.youtube.com/watch?v=5tYObQ3ocrw&yt:cc=on).
+
+## A Tela de Modelos {docsify-ignore}
+
+Você pode modificar modelos de cartões clicando no botão "Cartões..." dentro
+da tela de edição.
+
+Na parte superior esquerda está o modelo da frente, na parte inferior esquerda
+está o modelo do verso e no meio deles está a seção de estilo.
+
+No Anki, modelos são escritos em HTML, que é a linguagem que a internet
+usa pra escrever seus sites. A seção de estilos está em CSS, que é a
+linguagem usada pela internet para estilizar seus sites.
+
+Na direita está uma pré-visualização da frente e do verso do cartão
+selecionado no momento. Se você abriu essa janela enquanto estava adicionando
+uma nota, a pré-visualização vai ser baseada no atual conteúdo daquela nota.
+Se você abriu a janela a partir de Ferramentas → Gerenciar Tipos de Notas, o
+Anki vai mostrar o nome de cada campo entre parênteses no lugar do conteúdo.
+
+Na parte superior direita da janela existe um botão de Opções que te
```

Comment:
O que acha de suprimirmos o "te" nesse e noutros trechos? Para o contexto de um manual acredito ser mais adequado um texto mais impessoal: "que dá a opção de renomear"

Reply:
Sim, impessoalidade a um manual fica melhor.

Reply:
@mizerablebr Há de se notar (como dito no commit) o uso de "you" no original, mas acredito que mesmo assim seja válido suprimir o "te" como sugerido por você.